### PR TITLE
Eric http get

### PR DESCRIPTION
**Pull request by ESmith92**
Update http-get

```
-add timeout for curl calls that last longer than 2 mins.
-add check for failed curl call (returned false)
     -throws exception and returns information about the failed curl call
-add http get unit tests
        -add test to check that the http get is initalizable
        -add test to check for exception on bad curl call
               -checks that the exception message is about the curl call failure
```

### CODE REVIEW

File: src/commands/http-get.php [hunk 1]
```diff
@@ -11,6 +12,7 @@
 class HttpGet implements Command {
   private $curl;
   private $url;
+  private $seconds = 120;
```
Comment (rraub) [inline src/commands/http-get.php]: $seconds could be anything, how about renaming it to $timeout so its clear what the value is used for.

File: test/unit/http-get-unit-test.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+   * Test to check that httpget throws an exception on with bad url
+   *
+   * @expectedException Exception
+   * @expectedExceptionMessage error occured during curl exec. Additioanl info:
+   */
+  public function test_that_http_get_throws_exception_with_bad_website() {
+    $http_class = new SearchApi\Commands\HttpGet();
+    $http_class->setUrl( 'BadWebSite' );
+    $http_class->execute();
+  }
```
Comment (rraub) [inline test/unit/http-get-unit-test.php:40]: What about when you have a good url? you could test google.com if you want something that shouldn't be unavailable.

**Comment by rraub** [comment]
@ESmith92 don't forget about fixing the issues in the PR

**Comment by ESmith92** [comment]
@rraub what do you mean PR? Changing the seconds bar name?

**Comment by rraub** [comment]
PR = Pull Request

Yes the $seconds change and adding a "good" site to test with.

**Comment by ESmith92** [comment]
Alright, I'll do that in a few mins.

File: test/unit/http-get-unit-test.php [hunk 3]
```diff
@@ -35,7 +35,16 @@ public function test_that_http_get_throws_exception_with_no_url() {
    */
   public function test_that_http_get_throws_exception_with_bad_website() {
     $http_class = new SearchApi\Commands\HttpGet();
-    $http_class->setUrl( 'BadWebSite' );
-    $http_class->execute();
+    $http_class->setUrl( 'BadWebSite!!^#$%' );
+    var_dump($http_class->execute());
```
Comment (rraub) [inline test/unit/http-get-unit-test.php]: Why var_dump? I think you want to make an assertion or not output anything directly.

**Pull request merged**